code/game/turfs/closed/minerals.dm

```diff
@@ -659,7 +659,7 @@
 
 /turf/closed/mineral/gibtonite
 	mineralAmt = 1
-	icon_state = "rock_Gibtonite_inactive"
+	MAP_SWITCH(, icon_state = "rock_Gibtonite_inactive")
```

Comment:
```suggestion
	MAP_SWITCH(null, icon_state = "rock_Gibtonite_inactive")
```

this is the pattern you use elsewhere so we should keep it consistent. IMO we should just make a fully new macro instead of doing this but whatever

Reply:
I am switching between setting a var and not setting it. null would just runtime.
W 515 I could use :: here off parent_type, tho that might break downstreams